verification/curator-service/ui/cypress/integration/components/NewCaseForm.spec.ts

```diff
@@ -53,7 +53,7 @@ describe('New case form', function () {
         cy.visit('/cases/new');
         cy.contains('Create new COVID-19 line list case');
         cy.get('div[data-testid="caseReference"]').type('www.new-source.com');
-        cy.contains('li', 'www.new-source.com').click();
+        cy.get('input[name="caseReference.sourceName"]').click();
```

Comment:
do we want to keep a test where the source is clicked vs just entered?

Reply:
The other tests in this file still click the dropdown entry. There is also a test in BulkCaseForm that clicks the dropdown entry for a new source.